yk_face/face.py

```diff
@@ -70,6 +70,7 @@ def process(
     :raises:
         ValueError if image is not provided.
     """
+    configurations = configurations or []
```

Comment:
We should also raise an exception if processings is empty or null.

Reply:
This was moved to other function to avoid duplication.